src/cli/log.rs

```diff
@@ -65,7 +65,16 @@ where
     S: Subscriber + for<'span> LookupSpan<'span>,
 {
     let maybe_rust_log_directives = process.var_os("RUST_LOG").clone();
-    let has_ansi = process.stderr().is_a_tty();
+    let has_ansi = {
+        let env_override = process.var("RUSTUP_TERM_COLOR").map(|it| it.to_lowercase());
+        match env_override.as_deref() {
```

Comment:
Instead of doing `to_lowercase()`, consider doing `Ok(s) if s.eq_ignore_ascii_case("always")`.

Reply:
@djc This is copied from somewhere else in the codebase (also written by myself, as I've probably missed this `eq_ignore_ascii_case` thing), so I guess it'd be reasonable to change that part of the code according in a separate commit.